src/helm/benchmark/run_specs/speech_disorder_audio_run_specs.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+from helm.benchmark.adaptation.adapter_spec import AdapterSpec
+from helm.benchmark.metrics.common_metric_specs import audio_classification_metric_specs
+from helm.benchmark.metrics.metric import MetricSpec
+from helm.benchmark.run_spec import RunSpec, run_spec_function
+from helm.benchmark.run_specs.audio_run_specs import _get_multiple_choice_joint_adapter_spec
```

Comment:
Don't import private (leading `_`) methods. Either copy the method here or use the version from `common_adapter_specs`.